src/core/manager.c

```diff
@@ -561,13 +561,15 @@ static int manager_setup_signals(Manager *m) {
                         SIGRTMIN+4,  /* systemd: start poweroff.target */
                         SIGRTMIN+5,  /* systemd: start reboot.target */
                         SIGRTMIN+6,  /* systemd: start kexec.target */
+                        SIGRTMIN+7,  /* systemd: start renew.target */
 
                         /* ... space for more special targets ... */
 
                         SIGRTMIN+13, /* systemd: Immediate halt */
                         SIGRTMIN+14, /* systemd: Immediate poweroff */
                         SIGRTMIN+15, /* systemd: Immediate reboot */
                         SIGRTMIN+16, /* systemd: Immediate kexec */
+                        SIGRTMIN+17, /* systemd: Immediate renew */
 
                         /* ... space for one more immediate system state change ... */
```

Comment:
note: can remove this comment, there would be no space anymore here (the next one is `SIGRTMIN+18` which is used)